### ISSUE TITLE
How's the pre-trained semantic segmentation model trained on, Cityscape or KITTI?

### ISSUE THREAD

**Issue by ruili3** [TWJianNuo/EdgeDepth-Release#7]
Hi,

Thanks for your solid work and the datasets!

As you provided the pre-computed semantic labels for KITTI using method of Zhu _etal_ , I wonder if the semantic segmentation model you use is trianed on KITTI or Cityscape? Or a mixture of KITTI and Cityscape? I notice there is no pretrained models and training code for KITTI dataset on Zhu's github page. Thanks a lot! :D

**Comment by TWJianNuo** [comment]
According to their [paper](http://openaccess.thecvf.com/content_CVPR_2019/papers/Zhu_Improving_Semantic_Segmentation_via_Video_Propagation_and_Label_Relaxation_CVPR_2019_paper.pdf), it is pretrained on Cityscape and finetuned with kitti 200 semantics segmentation label.

**Comment by ruili3** [comment]
thank you a lot！

TWJianNuo <notifications@github.com> 于2020年11月11日周三 下午9:58写道：

**Comment by TWJianNuo** [comment]
Oh, It seems he removes pretrained model on kitti, strange. I used the model provided by the author.(but I do not know whether I still keep that.) In my memory, their model for kitti is pretrained using Cityscape's data plus Vista's data, then select hyper parameter for kitti under a 10-fold cross validation split of the 200 semantics data. Finally, they do a finetuning using kitti 200 semantics data with selected hyperparameter. By the way, tehre is going to be an additional kitti semantics dataset released, you can check this: http://panoptic.cs.uni-freiburg.de/

**Comment by ruili3** [comment]
Thank you for your reply~ I double checked the page and found the kitti
model on another [branch](
https://github.com/NVIDIA/semantic-segmentation/tree/sdcnet) of the
project. Thank you for the other link of the extra annotated dataset, it's
really awesome!

TWJianNuo <notifications@github.com> 于2020年11月12日周四 下午12:19写道：

**Comment by ruili3** [comment]
Thank you for your reply~ I double checked the page and found the kitti model on another [branch](https://github.com/NVIDIA/semantic-segmentation/tree/sdcnet) of the project. Thank you for the other link of the extra annotated dataset, it's really awesome!
